# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: caps/BasePrincipal.h
# Commit: 89e396b2896e
# Full Hash: 89e396b2896ed7fc168a5b4ec2f8ead05f5c6802
# Author: Andrea Marchesini <amarchesini@mozilla.com>
# Date: 2020-06-03 04:20:58
# Regressor Bug: 1638358
# File Overlap Count: 1
# Description:
#   Bug 1638358 - Cookie Schemeful Same-Site - part 5 - schemeful comparison, r=mayhemer
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75629
# ==============================================================================

diff -r 33ad5fa05209 -r 89e396b2896e caps/BasePrincipal.h
--- a/caps/BasePrincipal.h	Tue Jun 02 09:14:09 2020 +0000
+++ b/caps/BasePrincipal.h	Tue Jun 02 09:15:24 2020 +0000
@@ -122,6 +122,7 @@
   NS_IMETHOD GetIsContentPrincipal(bool* aResult) override;
   NS_IMETHOD GetIsExpandedPrincipal(bool* aResult) override;
   NS_IMETHOD GetIsSystemPrincipal(bool* aResult) override;
+  NS_IMETHOD GetScheme(nsACString& aScheme) override;
   NS_IMETHOD SchemeIs(const char* aScheme, bool* aResult) override;
   NS_IMETHOD IsURIInPrefList(const char* aPref, bool* aResult) override;
   NS_IMETHOD IsL10nAllowed(nsIURI* aURI, bool* aResult) override;